index.html

```diff
@@ -1,9 +1,12 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-<meta charset="UTF-8">
-‹meta http-equiv="X-UA-Compatible" content="IE=edge"> ‹meta name="viewport" content="width=device-width, initial-scale=1.0">
-‹title>Welcome to Potionfy</title>
+	<meta charset="UTF-8">
+	‹meta http-equiv="X-UA-Compatible" content="IE=edge">
+	‹meta name="viewport" content="width=device-width, initial-scale=1.0">
+	‹title>Welcome to Potionfy</title>
 </head>
-‹body> </body>
+‹body>
+	<h1>Welcome to Potionfy! Enjoy your stay.</h1>
```

Comment:
Could you remove the 'Enjoy your stay.' string and make the 'Welcome to Potionfy!' string red?|